### PR TITLE
Setting up sass

### PR DESCRIPTION
**Pull request by Ally4**
#### What does this PR do?

- This PR is for the setting up of sass

#### Description of Task to be completed?

- Installation of the package

- Creation of sass folder

- Creation of sass file

- Creation of css folder

- Creation of css file

#### How should this be manually tested?

- By cloning this repository

- By running `npm install` in the terminal

- By running `npm run compile:sass`

#### Any background context you want to provide?

- Installation of node-sass package

#### What are the relevant pivotal tracker stories?

- N/A

### CODE REVIEW

**Review by Mugema123 — APPROVED** [review]

**Review by TuyizeeAnastase — APPROVED** [review]

**Review by Ntwali-Josue — APPROVED** [review]
try to have one commit on your PR.

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "dev": "webpack-cli serve --mode development ",
     "build": "webpack --mode production",
     "start": "npm run build & serve -s ./dist ",
-    "coverage": "jest --coverage && cat ./coverage/lcov.info | coveralls"
+    "coverage": "jest --coverage && cat ./coverage/lcov.info | coveralls",
+    "compile:sass": "node-sass sass/index.scss css/index.css -w"
```
Comment (ceelogre) [inline package.json]: Add a rule in your weback file config for styles compilation. I don't know if you'd want to run this script on every development change.

**Review by ceelogre — CHANGES_REQUESTED** [review]
As a side note, `node-sass` is deprecated in favor of `dart-sass`

[read more](https://sass-lang.com/blog/libsass-is-deprecated)

File: package.json (hunk 1 above)
Comment (Ally4) [reply package.json]: I was thinking of putting the compile:sass in the script of dev, but let me work with the webpack configuration!.

Comment (Ally4) [reply package.json]: done!.

Comment (ceelogre) [reply package.json]: So do you still need this script?

**Review by ceelogre — APPROVED** [review]

**Pull request merged**